### DISCUSSION TITLE
Frontend Initiatives Meeting - March 16, 2021

### DISCUSSION THREAD

**Discussion by varl** [dhis2/notes#245] [Meetings] [answered]
When: March 16, 2021 @ 15:00 CET
Where: Remote

# Agenda

- Updates
  - General
    - TBD
    - Large push to App Platform last week
  - Working groups
    - PWA & Data Caching - call for task force members
    - App Runtime - field & paging warnings
    - App Hub Updates
         - Quick demo of publish command
    - TBD 
- Topics
  - PR Previews @paschalidi 
  - TBD
  - Anything else?  @dhis2/frontend

**Comment by martinkrulltott** [comment]
updates regarding using Netlify (or equivalent)?

**Comment by varl** [comment, marked as answer]
PR for minutes: https://github.com/dhis2/notes/pull/246

**Reply by varl** [reply]
Final: https://github.com/dhis2/notes/blob/master/decisions/2021/03/16-fe-initiatives.md
